test/ctr_userns.bats

```diff
@@ -15,26 +15,31 @@ function teardown() {
 }
 
 @test "ctr_userns run container" {
-	run crictl runp "$TESTDATA"/sandbox_config.json
-	echo "$output"
-	[ "$status" -eq 0 ]
-	pod_id="$output"
-	run crictl create "$pod_id" "$TESTDATA"/container_config_sleep.json "$TESTDATA"/sandbox_config.json
-	echo "$output"
-	[ "$status" -eq 0 ]
-	ctr_id="$output"
-	run crictl start "$ctr_id"
-	echo "$output"
-	[ "$status" -eq 0 ]
+	if test \! -e /proc/self/uid_map; then
+		skip "userNS not available"
+	fi
+	export CONTAINER_UID_MAPPINGS="0:100000:100000"
+	export CONTAINER_GID_MAPPINGS="0:200000:100000"
+
+	# Workaround for https://github.com/opencontainers/runc/pull/1562
```

Comment:
OK, it seems like you did something unholy here. I guess what happened is someone else (me?) modified the file (`test/ctr_userns.bats`), and the PR was merged while you were working on it. What this commit does looks like reverting to the older version of the file.

Yes, this was my commit ba3d36c003e5557f69a6a7453ba8b9d6a7575f83 (PR #4333).

Can you please fix this.

Reply:
fixed!